index.js

```diff
@@ -54,7 +68,13 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let total = 0
+  let average = total / movies.length
+  for (let i = 0; i < movies.length; i ++){
+    total += movies[i].imdbRating
+  }  return average
```

Comment:
Try to walk through our function here - before we loop as yourself what is `average`;

Also - when do we want to divide `total by average`?